File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/novaposhta/Areas/Datum.java
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.novaposhta.Areas;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Datum {

    @SerializedName("Ref")
    @Expose
    private String ref;
    @SerializedName("AreasCenter")
    @Expose
    private String areasCenter;
    @SerializedName("Description")
    @Expose
    private String description;

    public String getRef() {
        return ref;
    }

    public void setRef(String ref) {
        this.ref = ref;
    }

    public String getAreasCenter() {
        return areasCenter;
    }

    public void setAreasCenter(String areasCenter) {
        this.areasCenter = areasCenter;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return description;
    }
}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/novaposhta/Areas/Datum.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.novaposhta.Areas


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Datum {

    @SerializedName("Ref")
    @Expose
    var ref: String? = null
    @SerializedName("AreasCenter")
    @Expose
    var areasCenter: String? = null
    @SerializedName("Description")
    @Expose
    var description: String? = null

    override fun toString(): String {
        return description!!
    }
}